src/server/admin/server/admin_test.go

```diff
@@ -192,22 +193,52 @@ func RepoInfoToName(repoInfo interface{}) interface{} {
 	return repoInfo.(*pfs.RepoInfo).Repo.Name
 }
 
-// testExtractRestore effectively implements both TestExtractRestoreObjects
-// TestExtractRestoreNoObjects, as their logic is mostly the same
-func testExtractRestore(t *testing.T, testObjects bool) {
-	if os.Getenv("RUN_BAD_TESTS") == "" {
-		t.Skip("Skipping because RUN_BAD_TESTS was empty")
-	}
+// testExtractRestore effectively implements TestExtractRestoreObjects,
+// TestExtractRestoreNoObjects, TestExtractRestoreAuth, as their logic is mostly the same
+func testExtractRestore(t *testing.T, testObjects, testAuth bool) {
```

Comment:
Add optional auth support to the existing extract/restore tests. This way we don't have to duplicate the logic to build repos, pipelines, etc.